spacy_experimental/biaffine_parser/arc_predicter.pyx

```diff
@@ -53,30 +56,38 @@ def make_arc_predicter(
     model: Model,
     overwrite: bool,
     scorer: Optional[Callable],
+    senter: Optional[str],
+    max_tokens: int,
 ):
-    return ArcPredicter(nlp.vocab, model, name, overwrite=overwrite, scorer=scorer)
+    return ArcPredicter(nlp, model, name, max_tokens=max_tokens, overwrite=overwrite, scorer=scorer, senter=senter)
 
 
 class ArcPredicter(TrainablePipe):
     def __init__(
         self,
-        vocab: Vocab,
+        nlp: Language,
         model: Model,
         name: str = "arc_predicter",
         *,
+        max_tokens: int,
         overwrite: bool,
         scorer: Callable,
+        senter=None,
     ):
         self.name = name
         self.model = model
-        self.vocab = vocab
+        self.max_tokens = max_tokens
+        self.senter = nlp.get_pipe(senter) if senter is not None else None
```

Comment:
I kind of like `senter_name` because it makes it clear what it's referring to. Otherwise from the name `senter` I might have expected a full internal model? I don't have extremely strong feelings, though...

Reply:
Yeah, that's true, `_name` makes it clear that it's supposed to be a string, so I renamed it to `senter_name`.